_includes/docs_contents.html

```diff
@@ -1,41 +1,29 @@
 <div class="unit one-fifth hide-on-mobiles">
   <aside>
+    <!-- Only display the version links for versioned content -->
+    {% if page.collection != 'docs_enterprise' %}
     <ul>
       {% for collection in site.collections %}
-      <!-- Ignore "posts" and "docs", we only want versioned collections -->
+      <!-- Versioned collections are labeled docs_v4, docs_v5, etc -->
       <!-- prettier-ignore -->
-      {% if collection.label != "posts" and collection.label != "docs" %}
-      {% assign col_version = collection.label | split: "_" | last %}
+      {% assign col_version = collection.label | split: "_v" | last | plus: 0 %}
```

Comment:
This is a liquid hack to get a number as string then plus 0 to convert to integer